enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -319,6 +322,149 @@ the updates, and continue enforcement rollout only when the policy on previous c
                   operator: Exist
   ```
 
+#### Story 3
+
+As a system administrator, I want to specify a placement `DecisionStrategy` bound with policy sets, allowing rollouts to
+proceed across the set on a per-cluster basis and, optionally, in the order specified.
+
+- **Summary**
+
+  - Add an option in the policy set to specify an `orderPolicies` boolean, signaling to the
+    `governance-policy-propagator` controller to automatically inject policy dependencies down the list of policies in
+    the policy set when replicating policies (i.e. policy 1 is a dependency for policy 2, policy 2 is a dependency for
+    policy 3, and so on).
+  - Add `rolloutStrategy` and `rolloutStatus` to the `PolicySet`. For a policy set, the rollout is successful for a
+    cluster only after all policies in the set become compliant.
+  - The `rolloutStatus` would be an aggregated status across all clusters and policies. The aggregated root policy
+    status would be: `Progressing` (this would include `ToApply`), `Succeeded`, or `Failed`.
```

Comment:
Hmm. Interesting question. I could see it either way but I'm thinking it'd still be marked as Failed since the rollout failed on some clusters. For something like ProgressivePerGroup, setting MaxFailures would give a tolerance to allow the rollout to move on to the next group in the case of flaky cluster(s), so the aggregated status would stay in `Progressing` until then end, when it would wind up `Failed`.